src/box/vinyl.c

```diff
@@ -3195,6 +3196,7 @@ vy_gc_lsm(struct vy_lsm_recovery_info *lsm_info)
 
 /**
  * Delete unused run files stored in the recovery context.
+ * If there are no more runs, delete the root directory as well.
```

Comment:
Changes to this file are left from the previous patch version. Please revert them since they aren't needed anymore.

Reply:
Fixed.